views/partials/_mobile-navigation.njk

```diff
@@ -1,10 +1,18 @@
-<nav id="app-mobile-nav" class="app-mobile-nav js-app-mobile-nav" role="navigation">
+<nav id="app-mobile-nav" class="app-mobile-nav js-app-mobile-nav" role="navigation" aria-labelledby="app-mobile-navigation-heading">
+  <h2 class="govuk-visually-hidden" id="app-mobile-navigation-heading">Menu navigation</h2>
   <ul class="app-mobile-nav__list">
     {% for item in navigation %}
       <li>
         <div class="app-mobile-nav-subnav-toggler{% if path.startsWith(item.url) %} app-mobile-nav__subnav-toggler--active{% endif %}">
           <a class="govuk-link govuk-link--no-visited-state govuk-link--no-underline app-mobile-nav-subnav-toggler__link js-mobile-nav-subnav-toggler" href="/{{ item.url }}/">
-            {{ item.label }}
+            {% if item.items %}
+              <!-- When JavaScript is enabled, the menu links expand a section below with more content, so we should make them headings -->
+              <h3 class="app-mobile-nav-subnav-toggler__link-heading">{{ item.label }}</h3>
+              <span class="app-mobile-nav-subnav-toggler__link-text">{{ item.label }}</span>
```

Comment:
I wonder if we should consider wrapping the link text in an h3 using JS? It feels slightly more consistent with how we do this sort of thing elsewhere, and means we avoid duplicating the text in the source – which would be an issue if the CSS failed to load, for example.

Reply:
@36degrees Added here https://github.com/alphagov/govuk-design-system/pull/1744/commits/deab25c736679282d1a0e9f62c6ce2e243e464f5